top-tab/background.js

```diff
@@ -0,0 +1,5 @@
+chrome.runtime.onInstalled.addListener(function () {
+  chrome.storage.sync.set({color: '#01abaa'}, function () {
+    console.log("This color is turquoise, just like the epic stone.");
+  });
+});
```

Comment:
I think this is a feature that was started but not finished? (setting/loading the colour in local storage?)
Could use a comment to clarify, but I realize it's probably a time-crunch thing.